CareKit/CareCard/OCKCareCardViewController.m

```diff
@@ -497,4 +512,26 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     return cell;
 }
 
+#pragma mark - UIViewControllerPreviewingDelegate
+
+- (nullable UIViewController *)previewingContext:(id <UIViewControllerPreviewing>)previewingContext viewControllerForLocation:(CGPoint)location {
```

Comment:
@umerkhan-apple The delegate method calls for `nullable`. Also, what would I return if the if-statement is not satisfied and I want to cancel the peek? Returning `nil` is if the finger is in an invalid place to peek.